frontend/src/components/Forms/variances/VarianceSearchForm.js

```diff
@@ -0,0 +1,162 @@
+};
+
+const defaultProps = {
+  isAdvanceSearch: false,
+};
+
+const validate = (values) => {
+  const errors = {};
+  if (values.issue_date_min && values.issue_date_max) {
+    if (Date.parse(values.issue_date_min) > Date.parse(values.issue_date_max)) {
+      errors.issue_date_max = "Must be after issue date.";
+    }
+  }
+  if (values.expiry_date_min && values.expiry_date_max) {
+    if (Date.parse(values.expiry_date_min) > Date.parse(values.expiry_date_max)) {
+      errors.issue_date_max = "Must be after expiry date.";
+    }
+  }
+  return errors;
+};
+
+export class VarianceSearchForm extends Component {
+  handleReset = () => {
+    this.props.reset();
+    this.props.handleVarianceSearch({}, true);
+  };
+
+  render() {
+    return (
+      <Form layout="vertical" onSubmit={this.props.handleSubmit} onReset={this.handleReset}>
+        <Row gutter={6}>
+          <Col md={24} xs={24}>
+            <Field
+              id="search"
+              name="search"
+              component={renderConfig.FIELD}
+              placeholder="Search by code section, mine name, or mine number"
+            />
+          </Col>
+        </Row>
+        {this.props.isAdvanceSearch && (
+          <div>
+            <Row gutter={6}>
+              <Col md={24} xs={24}>
+                <Field
+                  id="compliance_code"
+                  name="compliance_code"
+                  placeholder="Select Compliance Code"
+                  component={renderConfig.MULTI_SELECT}
+                  data={this.props.complianceCodes}
+                />
+              </Col>
+            </Row>
+            <Row gutter={6}>
+              <Col md={12} xs={24}>
+                <Field
+                  id="issue_date_min"
+                  name="issue_date_min"
+                  placeholder="Select Min Issue Date"
+                  component={renderConfig.DATE}
+                  // validate={}
+                />
+              </Col>
+              <Col md={12} xs={24}>
+                <Field
+                  id="issue_date_max"
+                  name="issue_date_max"
+                  placeholder="Select Max Issue Date"
+                  component={renderConfig.DATE}
+                  // validate={}
+                />
+              </Col>
+            </Row>
+            <Row gutter={6}>
+              <Col md={12} xs={24}>
+                <Field
+                  id="expiry_date_min"
+                  name="expiry_date_min"
+                  placeholder="Select Min Expiry Date"
+                  component={renderConfig.DATE}
+                  // validate={}
+                />
+              </Col>
+              <Col md={12} xs={24}>
+                <Field
+                  id="expiry_date_max"
+                  name="expiry_date_max"
+                  placeholder="Select Max Expiry Date"
+                  component={renderConfig.DATE}
+                  // validate={}
```

Comment:
There are a few of these commented out that should be removed or updated to include validators